bin/jsonschema_suite

```diff
@@ -54,6 +54,17 @@ REMOTES = {
     "baseUriChange/folderInteger.json": {u"type": u"integer"},
     "baseUriChangeFolder/folderInteger.json": {u"type": u"integer"},
     "baseUriChangeFolderInSubschema/folderInteger.json": {u"type": u"integer"},
+    "locationIndependentIdentifier.json": {
+        "refToInteger": {
+            "$ref": "#foo"
+        },
+        "definitions": {
```

Comment:
it doesn't make much sense to me that this file has a schema on a properties of the root (like subSchemas.json) but the root also seems to be a schema, given the structure of `definitions`. it's probably better to put `refToInteger` in `definitions` too.